stylesheet.css

```diff
@@ -30,166 +41,328 @@ a:hover {
-  -ms-flex-align: center;
   align-items: center;
+  position: fixed;
 }
 
-/*-- navigation class --*/
-.nav-bar {
-  margin: 0;
+body > header > img {
+  max-width: 10rem;
+  padding-right: 2rem;
+}
+
+body > header > nav {
   display: flex;
-  align-items: center;
-  justify-content: center;
-  word-spacing: 1em;
+  justify-content: space-around;
+}
+
+#intro,
+#team,
+#projects,
+#open,
+#community,
+#sum-up_page,
+#sum-up_contact {
+  margin: auto;
+  width: 52rem;
 }
 
 /*-- intro seciton --*/
-.intro {
-  display: -webkit-flex;
-  display: -ms-flex;
-  -webkit-align-items: stretch;
-  -ms-flex-align: stretch;
+#intro {
+  display: flex;
+  margin-top: 2rem;
+  color: white;
+  padding-bottom: 4rem;
+  flex-direction: row;
+ }
+
+#intro > section {
+  padding: 1rem;
+  padding-bottom: 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  text-align: left;
   align-items: stretch;
+  word-wrap: break-word;
 }
 
-.intro-header {
-  -webkit-flex: 2 0 0; 
-  flex: 2 0 0;
-  align-items: center;
-  justify-content: center;
+#intro > header {
+
 }
 
-.intro-contact {
-  -webkit-flex: 1 0 0;
-  flex: 1 0 0;
-  padding: 15px;
+#intro > header > h1 {
+  font-weight: bolder;
+  font-size: 2em;
+  padding-bottom: 0.75rem;
+  text-align: left;
+}
+
+#intro > header > p {
+  text-transform: none;
+  text-align: left;
+}
+
+#intro > section > h2 {
+  text-transform: uppercase;
+  font-weight: normal;
+}
+
+#intro > section > p {
+  padding-top: 0.5rem;
+  font-size: 0.9em;
+  padding-bottom: 0.5rem;
+}
+
+#intro > section > aside {
+  text-align: right;
+  font-size: 0.6em;
 }
 
 .contact-box {
-  border-top-left-radius: 15px;
-  border-top-right-radius: 15px;
-  border-bottom-right-radius: 15px;
-  border-bottom-left-radius: 15px;
-  width: 200px; 
-  height: 25px;
+  border-top-left-radius: 1rem;
+  border-top-right-radius: 1rem;
+  border-bottom-right-radius: 1rem;
+  border-bottom-left-radius: 1rem;
+  width: 14rem;
+  height: 2rem;
+  padding: 0.5rem;
 }
 
 .submit-box {
   background-color: #73e600;
-  border-top-left-radius: 20px;
-  border-top-right-radius: 20px;
-  border-bottom-right-radius: 20px;
-  border-bottom-left-radius: 20px;
-  width: 200px; 
-  height: 25px;
+  border-top-left-radius: 1rem;
+  border-top-right-radius: 1rem;
+  border-bottom-right-radius: 1rem;
+  border-bottom-left-radius: 1rem;
+  width: 14rem;
+  height: 2rem;
 }
 
-.text-right {
+.intro-contact-email {
   text-align: right;
-}
-
-.submit-text {
-  font-size: 75%;
-  text-align: justify;
-  text-justify: inter-word;
-  text-align: center;
+  padding-bottom: 1rem;
 }
 
 .intro-text {
   text-align: center;
+  margin: auto;
+  width: 52rem;
+  padding-top: 2rem;
 }
 
 /*-- team section --*/
-.img-sec {
-  display: -webkit-flex;
-  display: -ms-flexbox;
+#team ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  text-align: center;
   display: flex;
-  -webkit-justify-content: center;
-  -ms-flex-pack: center;
+  flex-flow: wrap;
```

Comment:
`flex-flow` it's shorter version of `flex-wrap` and `flex-direction`. If you need to change only one of them - there is no need to use shorthand.